### PR TITLE
Cypress test. Create task without necessary arguments.

### PR DESCRIPTION
**Pull request by dvkruchinin**
### Motivation and context
Add Cypress test. Try to create task without necessary arguments.

### Checklist

- [x] I submit my changes into the `develop` branch
- [ ] I have added description of my changes into [CHANGELOG](https://github.com/opencv/cvat/blob/develop/CHANGELOG.md) file
- [ ] I have updated the [documentation](
  https://github.com/opencv/cvat/blob/develop/README.md#documentation) accordingly
- [ ] I have added tests to cover my changes
- [ ] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [ ] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [x] I have updated the license header for each file (see an example below)

```python
# Copyright (C) 2020 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

File: tests/cypress/integration/actions_tasks_objects/case_40_create_task_without_necessary_arguments.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+            cy.get('#cvat-create-task-button').click();
+        });
+
+        it('Try to create a task without any fields. A task is not created.', () => {
+            cy.get('.cvat-create-task-submit-section').click();
+            cy.get('.cvat-notification-create-task-fail').should('exist');
+            cy.closeNotification('.cvat-notification-create-task-fail');
+        });
+
+        it('Input a task name. Try to create a task without any fields. A task is not created.', () => {
```
Comment (bsekachev) [inline tests/cypress/integration/actions_tasks_objects/case_40_create_task_without_necessary_arguments.js:48]: Input a task name. A task is not created.

File: tests/cypress/integration/actions_tasks_objects/case_40_create_task_without_necessary_arguments.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+            cy.get('[id="name"]').type(taskName);
+            cy.get('.cvat-create-task-submit-section').click();
+            cy.get('.cvat-notification-create-task-fail').should('exist');
+            cy.closeNotification('.cvat-notification-create-task-fail');
+        });
+
+        it('Input task labels. Try to create a task without any fields. A task is not created.', () => {
```
Comment (bsekachev) [inline tests/cypress/integration/actions_tasks_objects/case_40_create_task_without_necessary_arguments.js:55]: Input task labels. A task is not created.

File: tests/cypress/integration/actions_tasks_objects/case_40_create_task_without_necessary_arguments.js [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+            cy.addNewLabel(labelName);
+            cy.get('.cvat-create-task-submit-section').click();
+            cy.get('.cvat-notification-create-task-fail').should('exist');
+            cy.closeNotification('.cvat-notification-create-task-fail');
+        });
+
+        it('Add some files. Try to create a task without any fields. A task created.', () => {
```
Comment (bsekachev) [inline tests/cypress/integration/actions_tasks_objects/case_40_create_task_without_necessary_arguments.js:62]: 'Add some files. A task created.

File: tests/cypress/integration/actions_tasks_objects/case_40_create_task_without_necessary_arguments.js [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+            cy.get('input[type="file"]').attachFile(archiveName, { subjectType: 'drag-n-drop' });
```
Comment (bsekachev) [inline tests/cypress/integration/actions_tasks_objects/case_40_create_task_without_necessary_arguments.js:63]: Does cypress guarantee exactly such order of tests when perform them?

**Review by bsekachev — CHANGES_REQUESTED** [review]

File: tests/cypress/integration/actions_tasks_objects/case_40_create_task_without_necessary_arguments.js (hunk 1 above)
Comment (dvkruchinin) [reply tests/cypress/integration/actions_tasks_objects/case_40_create_task_without_necessary_arguments.js]: Done.

File: tests/cypress/integration/actions_tasks_objects/case_40_create_task_without_necessary_arguments.js (hunk 2 above)
Comment (dvkruchinin) [reply tests/cypress/integration/actions_tasks_objects/case_40_create_task_without_necessary_arguments.js]: Done.

File: tests/cypress/integration/actions_tasks_objects/case_40_create_task_without_necessary_arguments.js (hunk 3 above)
Comment (dvkruchinin) [reply tests/cypress/integration/actions_tasks_objects/case_40_create_task_without_necessary_arguments.js]: Done.

File: tests/cypress/integration/actions_tasks_objects/case_40_create_task_without_necessary_arguments.js (hunk 4 above)
Comment (dvkruchinin) [reply tests/cypress/integration/actions_tasks_objects/case_40_create_task_without_necessary_arguments.js]: It groups are executed in a chain. Сan manipulate this using special constructs. https://docs.cypress.io/guides/core-concepts/writing-and-organizing-tests.html#The-order-of-hook-and-test-execution-is-as-follows

File: tests/cypress/integration/actions_tasks_objects/case_40_create_task_without_necessary_arguments.js [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+    after(() => {
+        cy.goToTaskList();
+        cy.getTaskID(taskName).then(($taskID) => {
+            cy.deleteTask(taskName, $taskID);
+        });
+    });
+
+    describe(`Testing "${labelName}"`, () => {
+        it('Go to create task page.', () => {
+            cy.get('#cvat-create-task-button').click();
```
Comment (bsekachev) [inline tests/cypress/integration/actions_tasks_objects/case_40_create_task_without_necessary_arguments.js:39]: Should be a part of before

Comment (dvkruchinin) [reply tests/cypress/integration/actions_tasks_objects/case_40_create_task_without_necessary_arguments.js]: Click to "Create new task" button moved to before.

**Review by bsekachev — APPROVED** [review]

**Pull request merged**